hugolib/site.go

```diff
@@ -784,6 +791,8 @@ func (s *Site) initializeSiteInfo() {
 		GoogleAnalytics:       viper.GetString("GoogleAnalytics"),
 		RSSLink:               s.permalinkStr(viper.GetString("RSSUri")),
 		BuildDrafts:           viper.GetBool("BuildDrafts"),
+		DisableSearchJSON:     viper.GetBool("DisableSearchJSON"),
+		SearchIndexLink:       viper.GetString("baseURL") + viper.GetString("searchuri"),
```

Comment:
That's what I've done before. I printed the URL in a template and got `http://localhost:1313/search/index.json/` instead of `http://localhost:1313/search.json`